source/core/utils/dns-ip-version.ts

```diff
@@ -16,7 +16,7 @@ export const isDnsIpVersion = (value: unknown): boolean => {
 };
 
 export const dnsIpVersionToFamily = (dnsIpVersion: DnsIpVersion): DnsIpFamily => {
-	if (isDnsIpVersion(dnsIpVersion)) {
+	if (dnsIpVersion in conversionTable) {
```

Comment:
The function above can be removed now, yup?

Reply:
No, because it's used in the assert inside the `normalizeArguments` function.

Reply:
Ok. Then can you change this if to use `isDnsIpVersion` back again and to define

```
export const isDnsIpVersion = (value: unknown): boolean => {
	return value in conversionTable;
};
```

?

Reply:
The `typeof` check is unnecessary I think.